sr_multi_moveit/ur10srh_moveit_config/config/ur10srh.srdf

```diff
@@ -74,24 +74,24 @@
         <joint name="rh_WRJ2" value="0" />
     </group_state>
     <group_state name="pack" group="right_hand">
-        <joint name="rh_FFJ1" value="0" />
+        <joint name="rh_FFJ1" value="0.35" />
```

Comment:
This was necessary for the biotacs. Maybe not the best idea to have it as default pack pose. It could be a `pack_biotacs`.